frontend/src/Components/Account/Register.js

```diff
@@ -97,15 +97,16 @@ export default function Register() {
           },
         })
         .then((resp) => {
-          console.log(resp.data.message); //later we can change log to notifications
+          toast.success(resp.data.message);
           navigate("/login");
           setLoading(false);
         })
-        .catch((err) => {
-          setFormError(true);
+        .catch(({ response }) => {
+          if (response) {
+            toast.error("Status " + response.status + " (" + response.statusText + "): " + JSON.stringify(response.data.detail));
```

Comment:
From a user perspective, the user need not have to see the status code and other server stuff right? He just needs to know why the error is caused.

I think  error detail alone is enough